pipelines/batch/setup_job.py

```diff
@@ -206,19 +221,47 @@ def main(
         "'"
     )
 
-    loc_abbrs = location_table.get_column("short_name").to_list()
-    locations_include = locations_include or loc_abbrs
-
-    # Always exclude the default locations
-    locations_exclude = locations_exclude or []
-    # Combine default exclusions with any additional exclusions
-    all_exclusions = list(set(DEFAULT_EXCLUDED_LOCATIONS + locations_exclude))
-
-    all_locations = [
-        loc
-        for loc in loc_abbrs
-        if loc not in all_exclusions and loc in locations_include
-    ]
+    print("")
+    print("=" * 58)
+    print(f"{'':<5} 🚀 pyrenew-hew: Azure Batch Job Submission 🚀")
+    print("=" * 58)
+    print(f"{'Job ID':<30}: {job_id}")
+    print(f"{'Pool ID':<30}: {pool_id}")
+    print(f"{'Model Family':<30}: {model_family}")
+    print(f"{'Model Letters':<30}: {model_letters}")
+    print(
+        f"{'Additional Forecast Letters':<30}: {additional_forecast_letters}"
+    )
+    print(f"{'Diseases':<30}: {', '.join(disease_list)}")
+    # Print locations, 5 per line for readability
+    for i, line in enumerate(range(0, len(all_locations), 5)):
+        locs = ", ".join(all_locations[line : line + 5])
+        print(f"{'Locations':<30}: {locs}" if i == 0 else f"{'':<30} {locs}")
+    print(f"{'Output Subdirectory':<30}: {output_subdir}")
+    print(f"{'Container Image':<30}: {container_image}")
+    print(f"{'Container Version':<30}: {container_image_version}")
+    print(f"{'Training Days':30}: {n_training_days}")
+    print(f"{'Exclude Last N Days':<30}: {exclude_last_n_days}")
+    print(f"{'Test Mode':<30}: {test}")
+    print(f"{'Dry Run':<30}: {dry_run}")
+    print("=" * 50)
```

Comment:
I added [`rich`](https://rich.readthedocs.io/en/latest/) as a dependency in https://github.com/CDCgov/pyrenew-hew/pull/588. Could make this a bit nicer / easier to write.